schema/codelists/tenderFinalStatus.csv

```diff
@@ -0,0 +1,3 @@
+Code,Title,Description
+cancelled,Cancelled,"The contracting process was cancelled by the buyer or the procuring entity (for example, because of a change in needs, insufficient funds, or technical or procedural errors) after the contracting documents (for example, procurement documents) were available to potential supppliers, but not later than the bid submission deadline."
```

Comment:
Missing 'complete'
```suggestion
complete,Complete,The bid submission deadline has passed.
cancelled,Cancelled,"The contracting process was cancelled by the buyer or the procuring entity (for example, because of a change in needs, insufficient funds, or technical or procedural errors) after the contracting documents (for example, procurement documents) were available to potential supppliers, but not later than the bid submission deadline."
```